app/src/main/java/com/example/robin/roomwordsample/fragments/NewToDoFragment.kt

```diff
@@ -148,7 +149,7 @@ class NewToDoFragment : Fragment() {
                     WorkManager.getInstance().enqueue(notifyManager)
                 }
                 replyIntent.putExtra("tag", tag)
-                wordViewModel.insert(Word(task, tm, tag))
+                wordViewModel.insert(Word(task, tm, tag,false,description))
```

Comment:
Add space after the comma and always remember to reformat the code after making changes in a file.